tests/test_spherical.py

```diff
@@ -0,0 +1,65 @@
+"""
+Tests ...
+"""
+from pytest import raises
+import numpy as np
+import spharpy.spherical as sh
+
+
+def test_renormalize():
+    sh_data = np.ones((4, 2))
+    current_norm = 'n3d'
+
+    # test invalid type
+    with raises(ValueError, match="Invalid normalization. Has to be 'sn3d', "
+                                  "'n3d', or 'maxN', but is wrong_norm"):
+        sh.renormalize(sh_data, 'acn', current_norm, 'wrong_norm', axis=0)
+
+    # test maxN
+    sh_data_renorm = sh.renormalize(sh_data.copy(), 'acn', current_norm,
+                                    'maxN', axis=0)
+    sh_data_ref = np.array([[np.sqrt(1 / 2), np.sqrt(1 / 2)],
+                            [np.sqrt(1 / 3), np.sqrt(1 / 3)],
+                            [np.sqrt(1 / 3), np.sqrt(1 / 3)],
+                            [np.sqrt(1 / 3), np.sqrt(1 / 3)]])
+
+    np.testing.assert_equal(sh_data_renorm,
+                            sh_data_ref)
+
+    # test sn3d
+    sh_data_renorm = sh.renormalize(sh_data.copy(), 'acn', current_norm,
+                                    'sn3d', axis=0)
+    sh_data_ref = np.array([[1 / np.sqrt(2 * 0 + 1), 1 / np.sqrt(2 * 0 + 1)],
+                            [1 / np.sqrt(2 * 1 + 1), 1 / np.sqrt(2 * 1 + 1)],
+                            [1 / np.sqrt(2 * 1 + 1), 1 / np.sqrt(2 * 1 + 1)],
+                            [1 / np.sqrt(2 * 1 + 1), 1 / np.sqrt(2 * 1 + 1)]])
+
+    np.testing.assert_equal(sh_data_renorm,
+                            sh_data_ref)
+    # test back to n3d
+    sh_data_renorm = sh.renormalize(sh_data.copy(), 'acn', current_norm, 'n3d',
+                                    axis=-2)
+    np.testing.assert_equal(sh_data_renorm,
+                            np.ones((4, 2)))
+
+
+def test_change_channel_convention():
```

Comment:
I would suggest to add `test_change_chanel_convention_errors` for testing if all errors are raised as expected